org.mixedrealitytoolkit.input/Interactors/GazePinch/GazePinchInteractor.cs

```diff
@@ -70,12 +70,12 @@ public class GazePinchInteractor :
 
         [SerializeField]
         [Tooltip("The interactor we're using to query potential gaze pinch targets")]
-        private XRBaseControllerInteractor dependentInteractor;
+        private XRBaseInteractor dependentInteractor;
```

Comment:
This looks like a breaking change.   We probably want to hold off on this change until the next major release.

Reply:
As mentioned above only the `IXRInteracator` implementations (namely `GetValidTargets` and the casting of extended interfaces) are used. But if someone is extending this class and depending on implementations specific to `XRBaseControllerInteractor`, then, yes, this change could cause issues. Otherwise, functionally, this change would not make any difference; it works on both XRI 2.5.2 and XRI 3.0.0-pre.1.